<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="player_img_lb">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>220</width>
      <height>320</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>371</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Eras Demi ITC</family>
      <pointsize>30</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Player</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="player_name_lb">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>471</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Eras Demi ITC</family>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Super Hero Name</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>140</y>
      <width>241</width>
      <height>321</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>220</width>
       <height>321</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QPushButton" name="player_intel_btn">
          <property name="font">
           <font>
            <family>Eras Demi ITC</family>
            <pointsize>20</pointsize>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>Intelligence</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="player_str_btn">
          <property name="font">
           <font>
            <family>Eras Demi ITC</family>
            <pointsize>20</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Strength</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="player_spd_btn">
          <property name="font">
           <font>
            <family>Eras Demi ITC</family>
            <pointsize>20</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Speed</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="player_dura_btn">
          <property name="font">
           <font>
            <family>Eras Demi ITC</family>
            <pointsize>20</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Durability</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="player_pwr_btn">
          <property name="font">
           <font>
            <family>Eras Demi ITC</family>
            <pointsize>20</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Power</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="player_combat_btn">
          <property name="font">
           <font>
            <family>Eras Demi ITC</family>
            <pointsize>20</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Combat</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="player_intel_lb">
          <property name="font">
           <font>
            <family>Eras Demi ITC</family>
            <pointsize>20</pointsize>
           </font>
          </property>
          <property name="text">
           <string>###</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="player_str_lb">
          <property name="font">
           <font>
            <family>Eras Demi ITC</family>
            <pointsize>20</pointsize>
           </font>
          </property>
          <property name="text">
           <string>###</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="player_spd_lb">
          <property name="font">
           <font>
            <family>Eras Demi ITC</family>
            <pointsize>20</pointsize>
           </font>
          </property>
          <property name="text">
           <string>###</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="player_dura_lb">
          <property name="font">
           <font>
            <family>Eras Demi ITC</family>
            <pointsize>20</pointsize>
           </font>
          </property>
          <property name="text">
           <string>###</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="player_pwr_lb">
          <property name="font">
           <font>
            <family>Eras Demi ITC</family>
            <pointsize>20</pointsize>
           </font>
          </property>
          <property name="text">
           <string>###</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="player_combat_lb">
          <property name="font">
           <font>
            <family>Eras Demi ITC</family>
            <pointsize>20</pointsize>
           </font>
          </property>
          <property name="text">
           <string>###</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QLabel" name="ai_img_lb">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>140</y>
      <width>220</width>
      <height>320</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="ai_name_lb">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>80</y>
      <width>471</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Eras Demi ITC</family>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Super Hero Name</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>140</y>
      <width>241</width>
      <height>321</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>220</width>
       <height>321</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="label_8">
          <property name="font">
           <font>
            <family>Eras Demi ITC</family>
            <pointsize>20</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Intelligence</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_7">
          <property name="font">
           <font>
            <family>Eras Demi ITC</family>
            <pointsize>20</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Strength</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_6">
          <property name="font">
           <font>
            <family>Eras Demi ITC</family>
            <pointsize>20</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Speed</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_5">
          <property name="font">
           <font>
            <family>Eras Demi ITC</family>
            <pointsize>20</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Durability</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_4">
          <property name="font">
           <font>
            <family>Eras Demi ITC</family>
            <pointsize>20</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Power</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label">
          <property name="font">
           <font>
            <family>Eras Demi ITC</family>
            <pointsize>20</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Combat</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QLabel" name="ai_intel_lb">
          <property name="font">
           <font>
            <family>Eras Demi ITC</family>
            <pointsize>20</pointsize>
           </font>
          </property>
          <property name="text">
           <string>###</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="ai_str_lb">
          <property name="font">
           <font>
            <family>Eras Demi ITC</family>
            <pointsize>20</pointsize>
           </font>
          </property>
          <property name="text">
           <string>###</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="ai_spd_lb">
          <property name="font">
           <font>
            <family>Eras Demi ITC</family>
            <pointsize>20</pointsize>
           </font>
          </property>
          <property name="text">
           <string>###</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="ai_dura_lb">
          <property name="font">
           <font>
            <family>Eras Demi ITC</family>
            <pointsize>20</pointsize>
           </font>
          </property>
          <property name="text">
           <string>###</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="ai_pwr_lb">
          <property name="font">
           <font>
            <family>Eras Demi ITC</family>
            <pointsize>20</pointsize>
           </font>
          </property>
          <property name="text">
           <string>###</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="ai_combat_lb">
          <property name="font">
           <font>
            <family>Eras Demi ITC</family>
            <pointsize>20</pointsize>
           </font>
          </property>
          <property name="text">
           <string>###</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>20</y>
      <width>361</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Eras Demi ITC</family>
      <pointsize>30</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Computer</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="player_hand_lb">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>20</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Eras Demi ITC</family>
      <pointsize>30</pointsize>
     </font>
    </property>
    <property name="text">
     <string>###</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="ai_hand_lb">
    <property name="geometry">
     <rect>
      <x>1080</x>
      <y>20</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Eras Demi ITC</family>
      <pointsize>30</pointsize>
     </font>
    </property>
    <property name="text">
     <string>###</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>140</y>
      <width>201</width>
      <height>321</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <item>
      <widget class="QLabel" name="stat_lb">
       <property name="font">
        <font>
         <family>Eras Demi ITC</family>
         <pointsize>22</pointsize>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_9">
       <property name="font">
        <font>
         <family>Eras Demi ITC</family>
         <pointsize>30</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Winner:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignHCenter|Qt::AlignTop</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="win_lb">
       <property name="font">
        <font>
         <family>Eras Demi ITC</family>
         <pointsize>24</pointsize>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignHCenter|Qt::AlignTop</set>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_10">
         <property name="font">
          <font>
           <family>Eras Demi ITC</family>
           <pointsize>24</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Kitty:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="kitty_lb">
         <property name="font">
          <font>
           <family>Eras Demi ITC</family>
           <pointsize>24</pointsize>
          </font>
         </property>
         <property name="text">
          <string>###</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTop|Qt::AlignTrailing</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="victory_lb">
       <property name="font">
        <font>
         <family>Eras Demi ITC</family>
         <pointsize>30</pointsize>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignHCenter|Qt::AlignTop</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <zorder>frame</zorder>
   <zorder>player_img_lb</zorder>
   <zorder>label_2</zorder>
   <zorder>player_name_lb</zorder>
   <zorder>ai_img_lb</zorder>
   <zorder>ai_name_lb</zorder>
   <zorder>frame_2</zorder>
   <zorder>label_3</zorder>
   <zorder>player_hand_lb</zorder>
   <zorder>ai_hand_lb</zorder>
   <zorder>win_lb</zorder>
   <zorder>label_9</zorder>
   <zorder>label_10</zorder>
   <zorder>kitty_lb</zorder>
   <zorder>victory_lb</zorder>
   <zorder>stat_lb</zorder>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
